app/services/onboarding_csv_reader.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'csv'
+require_relative 'onboarding_csv_parser'
+
+#--------------------------
+#
+# @class OnboardingCsvReader
+#
+# @desc Responsibility: Import Data in an Onboarding CSV file.  Create
+#   objects as needed if they don't already exist.
+#
+# @author Ashley Engelund (ashley.engelund@gmail.com  weedySeaDragon @ github)
+# @date   11/01/20
+#
+#--------------------------
+#
+class OnboardingCsvReader
+  def self.import(full_filename)
+    raise ArgumentError, 'Must provide a filename' if full_filename.blank?
+
+    json = String.new  # TODO: rubocop flags this, but I don't know how else to initialize it.
```

Comment:
@katelovescode 
This is what Rubocop is complaining about.
I'm not sure how else to initialize this empty String.  (No doubt I'm having a brain fart.) Halp!

Reply:
```
json = ''
```
No need to initialize it as a string literal

Reply:
Nope -- that doesn't work.  That gives you a `FrozenError`:
```
 FrozenError:
       can't modify frozen String
```

Reply:
Hence my puzzlement with Rubocop.

Reply:
Okay, this is because of the default frozen string literal setting that's recommended.  I'm near positive it's a preferred way of handling strings for performance reasons.  Could you do this instead:

```
  def self.import(full_filename)
    raise ArgumentError, 'Must provide a filename' if full_filename.blank?
    File.open(full_filename, 'r') { |csv_file| OnboardingCSVParser.parse(csv_file.read) }
  end
```

Since Ruby methods return the evaluation of the last line called?

Reply:
Yup - that's why.  (It's a 'performance' cop.)
I've already refactored to a slightly different way that is passing.
Let me know what you think & if you want me to use your suggestions.